Dockerfile

```diff
@@ -1093,7 +1099,9 @@ RUN set -x; \
 	# For Widgets extension
 	&& mkdir -p $MW_ORIGIN_FILES/extensions/Widgets \
 	&& mv $MW_HOME/extensions/Widgets/compiled_templates $MW_ORIGIN_FILES/extensions/Widgets/ \
-	&& ln -s $MW_VOLUME/extensions/Widgets/compiled_templates $MW_HOME/extensions/Widgets/compiled_templates
+	&& ln -s $MW_VOLUME/extensions/Widgets/compiled_templates $MW_HOME/extensions/Widgets/compiled_templates \
+	# Modify /etc/bash.bashrc (loaded for `docker-compose exec web bash` command)
+	&& echo '. /etc/profile.d/load-env-vars.sh' >> /etc/bash.bashrc
```

Comment:
would adding this to `~/.bashrc` here instead of `/etc/bash.bashrc` fix the issue Alexey pointed out on the task?